Jenkinsfile

```diff
@@ -53,6 +61,38 @@ pipeline {
 
       steps {
         sh 'cd ci && summon ./jenkins_build'
+        script {
+          TAG = "${env.TAG}"
+        }
+      }
+    }
+
+    stage('Scan images for vulnerabilities') {
+      parallel {
+        /*
+         * A scan of the conjur image is skipped since this image is scanned
+         * vulnerabilities for builds in the cyberark/conjur repository.
+         */
+        stage('Scan deployer image') {
+          steps {
+            scanAndReport("gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:${TAG}", "CRITICAL")
+          }
+        }
+        stage('Scan tester image') {
+          steps {
+            scanAndReport("gcr.io/conjur-cloud-launcher-onboard/cyberark/tester:${TAG}", "HIGH")
+          }
+        }
+        stage('Scan nginx image') {
+          steps {
+            scanAndReport("gcr.io/conjur-cloud-launcher-onboard/cyberark/nginx:${TAG}", "CRITICAL")
+          }
+        }
+        stage('Scan postgres image') {
+          steps {
+            scanAndReport("gcr.io/conjur-cloud-launcher-onboard/cyberark/postgres:${TAG}", "CRITICAL")
+          }
+        }
```

Comment:
Do we need `conjur` image here too or is that part of deployer?

Reply:
The conjur image is being scanned when it gets built as part of the cyberark/conjur repo builds, so the recommendation (from Andy) was to skip the scan here. I put a comment in the Jenkinsfile, line 73.
I'll add a comment on this in this PR's description.

Reply:
Ah good catch - I totally missed that comment!